scripts/python3.8/linux/build_in_docker.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+#
+# This script is used in the docker container to create a Cura AppImage.
+#
+
+set -e
+
+# Get where this script resides
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+ROOT_DIR="${SCRIPT_DIR}/../../.."
+
+# Make sure that cura-build-environment is present
+if [[ -z "${CURA_BUILD_ENV_PATH}" ]]; then
+    echo "CURA_BUILD_ENV_PATH is not defined. Cannot find the installed cura build environment."
+    exit 1
+fi
+
+# Make sure that a directory for saving the resulting AppImage exists
+CURA_BUILD_OUTPUT_DIR="${CURA_BUILD_OUTPUT_DIR:-/home/ultimaker/src/output}"
+if [[ ! -d "${CURA_BUILD_OUTPUT_DIR}" ]]; then
+    mkdir -p "${CURA_BUILD_OUTPUT_DIR}"
+fi
+
+# Set up Cura build configuration in environment variables
+export CURA_BRANCH_OR_TAG="${CURA_BRANCH_OR_TAG:-master}"
+export URANIUM_BRANCH_OR_TAG="${URANIUM_BRANCH_OR_TAG:-master}"
+export CURAENGINE_BRANCH_OR_TAG="${CURAENGINE_BRANCH_OR_TAG:-master}"
+export CURABINARYDATA_BRANCH_OR_TAG="${CURABINARYDATA_BRANCH_OR_TAG:-master}"
+export FDMMATERIALS_DATA_BRANCH_OR_TAG="${FDMMATERIALS_BRANCH_OR_TAG:-master}"
+export LIBCHARON_BRANCH_OR_TAG="${LIBCHARON_BRANCH_OR_TAG:-master}"
+
+export CURA_VERSION_MAJOR="${CURA_VERSION_MAJOR:-9}"
+export CURA_VERSION_MINOR="${CURA_VERSION_MINOR:-9}"
+export CURA_VERSION_PATCH="${CURA_VERSION_PATCH:-9}"
+export CURA_VERSION_EXTRA="${CURA_VERSION_EXTRA:-}"
+
+export CURA_BUILD_TYPE="${CURA_BUILD_TYPE}"
+export CURA_NO_INSTALL_PLUGINS="${CURA_NO_INSTALL_PLUGINS}"
+
+export CURA_CLOUD_API_ROOT="${CURA_CLOUD_API_ROOT:-https://api.ultimaker.com}"
+export CURA_CLOUD_API_VERSION="${CURA_CLOUD_API_VERSION:-1}"
+export CURA_CLOUD_ACCOUNT_API_ROOT="${CURA_CLOUD_ACCOUNT_API_ROOT:-https://account.ultimaker.com}"
+export CURA_MARKETPLACE_ROOT="${CURA_MARKETPLACE_ROOT:-https://marketplace.ultimaker.com}"
+export CURA_DIGITAL_FACTORY_URL="${CURA_DIGITAL_FACTORY_URL:-https://digitalfactory.ultimaker.com}"
+
+export CURA_ENABLE_DEBUG_MODE="${CURA_ENABLE_DEBUG_MODE:-ON}"
+export CURA_ENABLE_CURAENGINE_EXTRA_OPTIMIZATION_FLAGS="${CURA_ENABLE_CURAENGINE_EXTRA_OPTIMIZATION_FLAGS:-ON}"
```

Comment:
As far as I know this variable is always ON now and doesn't need to be a parameter.

Reply:
Sure, I am also fine with that.